client/components/parts/ButtonField.jsx

```diff
@@ -33,13 +34,56 @@ export default class ButtonField extends Component{
       'facebook',
       'twitter',
       'instagram'
-    ]
+    ],
+    typeTemplates: {
+      link: {
+        label: 'Link URL',
+        placeholder: 'http://www.somewhere.com'
+      },
+      email: {
+        label: 'Email Address',
+        preprefix: 'mailto:',
+        placeholder: 'person@somewhere.com'
```

Comment:
why a preprefix but no prefix? 
Maybe better language could be chosen for this part of the templates?